third_party/CMakeLists.txt

```diff
@@ -56,6 +56,31 @@ target_compile_options(
     -O3 -Wno-sign-compare
 )
 
+add_library(
+    cqf
+
+    cqf/src/cqf2.cpp
+    cqf/src/cqf4.cpp
+    cqf/src/cqf8.cpp
+    cqf/src/cqf16.cpp
+    cqf/src/cqf32.cpp
+)
+
+target_compile_options(
+    cqf
+    PRIVATE
+    -O3
```

Comment:
you shouldn't have to set this, the CMAKE_BUILD_TYPE will take care of this. Any reason to force -O3 for all build types?

Reply:
Removed.